### PR TITLE
added the frame types for filled circle and hollow circle

### PR DESCRIPTION
**Pull request by lldiegon**
Added the frames for writing a filled circle & a hollow circle to the display.

### CODE REVIEW

File: code/headers/frame_types.hpp [hunk 1]
```diff
@@ -225,6 +227,57 @@ namespace r2d2 {
...
+     *
+     * Currently we can't fill the bigger screens. When the
+     * extended frames are here the position and width/height
+     * will change to a uint16_t to support the bigger screens.
+     *
+     * Display wiki:
+     * https://github.com/R2D2-2019/R2D2-2019/wiki/Display
+     */
+    R2D2_PACK_STRUCT
+    struct frame_display_hollow_circle_s {
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp]: - Are you going to create a cursor and non-cursor version of each display frame? Currently a cursor is useless when drawing a circle.
 - Maybe a boolean "filled" would be more logical, as that is the only difference between these frames

**Review by LRstudentHU — CHANGES_REQUESTED** [review]

File: code/headers/frame_types.hpp (hunk 1 above)
Comment (lldiegon) [reply code/headers/frame_types.hpp]: I'm not going to do anything with the cursor for drawing a circle. Adding a 'filled' boolean is a good idea, I will make the changes and then commit again. Thank you.

File: code/headers/frame_types.hpp [hunk 2]
```diff
@@ -225,6 +226,35 @@ namespace r2d2 {
         char characters[247];
     };
 
+    /**
+     * Struct to set a circle on a display. This fills a
+     * circle with the color specified.
+     *
+     * Currently we can't fill the bigger screens. When the
+     * extended frames are here the position and width/height
```
Comment (itzandroidtab) [inline code/headers/frame_types.hpp]: Extended frames are supported atm. But currently it doesn't look like the bigger screens are gonna be implemented. So there is no need to change the types of the position or the width/height. So can you remove the part of changing the type to a uint16_t when the extended frames are here.

Comment (itzandroidtab) [reply code/headers/frame_types.hpp]: Maybe a todo: 

* Update the display rectangle as well in a new PR. 
* Add a filled bool to that as well.

**Review by itzandroidtab — CHANGES_REQUESTED** [review]
See extended frame comment

File: code/headers/frame_types.hpp (hunk 1 above)
Comment (LRstudentHU) [reply code/headers/frame_types.hpp]: Why not, actually? It seems a bit weird that there are two API's but some actions support on or the other, while others support both.

Comment (lldiegon) [reply code/headers/frame_types.hpp]: Fair enough, I submitted a new commit.

**Review by LRstudentHU — APPROVED** [review]

**Review by itzandroidtab — APPROVED** [review]

**Pull request merged**